src/ir_Mitsubishi.cpp

```diff
@@ -452,20 +454,33 @@ void IRMitsubishiAC::setMode(const uint8_t mode) {
       return;
   }
   _.Mode = mode;
+  // iSave10C can only be on in Heat mode.
+  if (mode != kMitsubishiAcHeat) {
+      setiSave10C(false);
+  }
 }
 
 /// Set the iSave10C (i-SAVE) mode of the A/C.
 /// @param[in] on true, the setting is on. false, the setting is off.
-/// Normal minimum temp is 16C, and i-SAVE mode works as a gate to enable AC to use 10C as setting. However, when Remote control shows 10C, it still emits 16C on the "Temp" bits, 
-/// and instead it uses other bits to indicate a target temp of 10C. Slightly strange, but I guess it's to keep compatibility to systems without i-SAVE.
-/// i-SAVE only has this 10C functionality when the AC is already in Heat mode. In all other modes, minimum temp is 16C.
-/// I have found no other difference between normal Heat mode and i-SAVE other than the ability to go to 10C.
-/// In this implementation, i-SAVE mode is ONLY used to enable the AC temperature setting to 10C. Therefore "Temp" is set to 16 disregarding what the remote shows, and mode is set to Heat.
+/// Normal minimum temp is 16C, and i-SAVE mode works as a gate to enable AC
```

Comment:
`/// @note Normal minimum temp is 16C, and i-SAVE mode works as a gate to enable AC`

Reply:
Not sure what you are requesting here?

Reply:
done